subsys/modem/modem_cmux.c

```diff
@@ -652,6 +656,19 @@ static void modem_cmux_on_frame(struct modem_cmux *cmux)
 	}
 }
 
+static void modem_cmux_drop_frame(struct modem_cmux *cmux)
+{
+	LOG_WRN("Dropped frame");
+	cmux->receive_state = MODEM_CMUX_RECEIVE_STATE_SOF;
+
+#if CONFIG_MODEM_CMUX_LOG_LEVEL_DBG
```

Comment:
Better to use `#if defined(CONFIG_MODEM_CMUX_LOG_LEVEL_DBG)` as that will not give error if the define does not exists.